python/lsst/meas/deblender/deblend.py

```diff
@@ -467,67 +470,67 @@ class MultibandDeblendConfig(pexConfig.Config):
-                                            "iterations will fit the peaks, then positions are"
-                                            "updated on every 'skipTranslation' iterations."))
-    skipTranslation = pexConfig.Field(dtype=int, default=10,
-                                      doc=("If fitPositions is True, the first 'requireTranslations'"
-                                           "iterations will fit the peaks, then positions are"
-                                           "updated on every 'skipTranslation' iterations."))
+    refinementSkip = pexConfig.Field(dtype=int, default=10,
+                                      doc=("If fitPositions is True, the positions and box sizes are"
+                                           "updated on every 'refinementSkip' iterations."))
     translationMethod = pexConfig.Field(dtype=str, default="default",
                                         doc=("Method to use for fitting translations."
                                              "Currently 'default' is the only available option,"
                                              "which performs a linear fit, but it is possible that we"
                                              "will use galsim or some other method as a future option"))
+    edgeFluxThresh = pexConfig.Field(dtype=float, default=1.0,
+                                     doc=("Boxes are resized when the flux at an edge is "
+                                          "> edgeFluxThresh * bg_rms"))
+    exactLipschitz = pexConfig.Field(dtype=bool, default=False,
+                                     doc=("Calculate exact Lipschitz constant in every step"
+                                          "(exact_lipschitz is True) or only calculate the"
+                                          "Lipschitz constant with significant changes in A,S"
+                                          "(exact_lipschitz is False)"))
+    stepSlack = pexConfig.Field(dtype=float, default=0.2,
+                                doc=("Slack to use when updating the step size in each iteration of"
+                                     "the bSDMM minimization"))
 
     # Constraints
-    multipleComponents = pexConfig.Field(dtype=bool, default=False,
-                                         doc=("Whether or not to fit multiple components for each peak"))
-    constraints = pexConfig.Field(dtype=str, default="S",
-                                  doc=("List of constraints to use for each object"))
+    constraints = pexConfig.Field(dtype=str, default="1+SM",
+                                  doc=("List of constraints to use for each object."
+                                       "Current options are all used by default:\n"
+                                       "S: symmetry\n"
+                                       "M: monotonicity\n"
+                                       "1: normalized SED to unity"
+                                       "+: non-negative morphology"))
```

Comment:
Why is this a string? And if it should be a string, why call it a list? And does ordering matter? If you are expecting more future constraints adding an explicit (comma) separator might be more future-proof.

Reply:
Made this a list and made it clear that order does not matter